_data/simple-icons.json

```diff
@@ -15,6 +15,11 @@
             "hex": "00A98F",
             "source": "https://about.me/assets"
         },
+        {
+            "title": "ABB Robotstudio",
+            "hex": "EC1B24",
```

Comment:
This seems to be the brand colour for ABB itself, what makes you choose it as the brand color for the RobotStudio?

Reply:
We could also go with `#FF9E0F`. Maybe that's better, because it's a color from the logo.

By the way a high resolution logo can be found [at the MS Store](https://www.microsoft.com/en-us/p/robotstudio-holographic).